### ISSUE TITLE
no response from alpaca-electron

### ISSUE THREAD

**Issue by glowingkitty** [ItsPi3141/alpaca-electron#39]
Tried the macOS x86 version. It starts. But when loading the Alpaca model and entering a message, it never responds. In other words: can't make it work on MacOS.

Using MacOS 13.2 on an MacBook Pro M1 (2020).

**Comment by Acidbuk** [comment]
Yeah, doing the same here. was working fine last night. loaded it this morning and it just won't respond. its like it is not loading the model.

**Comment by ItsPi3141** [comment]
> Tried the macOS x86 version. It starts. But when loading the Alpaca model and entering a message, it never responds. In other words: can't make it work on MacOS.
> 
> Using MacOS 13.2 on an MacBook Pro M1 (2020).

Why are you using the x64 version? It runs really slow on ARM64 Macs. Use the ARM64 version instead. Or does the ARM64 build not work?

**Comment by thelamedia** [comment]
> > Tried the macOS x86 version. It starts. But when loading the Alpaca model and entering a message, it never responds. In other words: can't make it work on MacOS.
> > Using MacOS 13.2 on an MacBook Pro M1 (2020).
> 
> Why are you using the x64 version? It runs really slow on ARM64 Macs. Use the ARM64 version instead. Or does the ARM64 build not work?

The ARM64 version is broken and cannot be loaded.  I am having all the same issues as above.

**Comment by ItsPi3141** [comment]
I don't have an arm64 Mac. But I guess it's either an arm64 thing or just the M1 chipset. Its pretty quick on my Intel Mac virtual machine.
